### PR TITLE
Add storyCard component UI

### LINKED ISSUE #115: Status site display ongoing features
Design: [Figma Link](https://www.figma.com/file/YamxCndtBmO9UK1JVFp1Ui/Status-New-Features?node-id=0%3A1)

Let's start by creating this component, writing its tests and adding it into the storybook

### PR DESCRIPTION
**Pull request by Cartmanishere**
Closes #115 

Design Link: https://www.figma.com/file/YamxCndtBmO9UK1JVFp1Ui/Status-New-Features?node-id=0%3A1

Component:

Screen Shot 2021-06-22 at 11 13 52 PM

### CODE REVIEW

File: src/components/storyCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    <div className={styles.container}>
+      <div className={styles.task_info}>
+        <div className={styles.title}>
+          {title}
+        </div>
+        <div className={styles.description}>
+          {description}
+        </div>
+        <div className={styles.subtasks}>
+          {subtasks.map((task) => (<Task title={task.title} status={task.status} />))}
```
Comment (sumitd94) [inline src/components/storyCard/index.tsx:50]: missing `key` attribute

File: src/components/storyCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                  src={owners.frontend.img}
+                  alt={owners.frontend.name}
+                />
+              )
+              : (
+                <div className={styles.to_be_decided}>
+                  <p>TBD</p>
+                </div>
+              )}
+          </div>
```
Comment (sumitd94) [inline src/components/storyCard/index.tsx:117]: the `owner_info` div has almost a similar structure and it's repeating 3 times.
I think we can create a function `renderOwnerInfo` and pass the necessary data and that function will return this `div structure`.
What say?

**Review by sumitd94 — CHANGES_REQUESTED** [review]
small changes requested, please

File: src/components/storyCard/index.tsx (hunk 1 above)
Comment (Cartmanishere) [reply src/components/storyCard/index.tsx]: Done

File: src/components/storyCard/index.tsx (hunk 2 above)
Comment (Cartmanishere) [reply src/components/storyCard/index.tsx]: Yeah, makes sense. Extracted the common parts and converted it to a function. Thanks.

File: src/components/storyCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+      <p>TBD</p>
+    </div>
+  );
+
+  if (owner) {
+    ownerEl = (
+      <img
+        className={styles.owner_image}
+        src={owner.img}
+        alt={owner.name}
```
Comment (whyDontI) [inline src/components/storyCard/index.tsx:48]: Put dummy image `onError`

**Review by whyDontI — CHANGES_REQUESTED** [review]
Requesting a small changes,
- We try to follow `camelCase` for all variables and classNames.
It'll be great if you can convert classNames to `camelCase`

File: src/components/storyCard/storyCard.module.scss [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    justify-content: space-between;
+}
+
+.toBeDecided > p {
+    margin: 5px;
+}
+
+.dueDate {
+    margin-left: 10px;
+}
```
Comment (sumitd94) [inline src/components/storyCard/storyCard.module.scss:109]: One question, is your tab size set to 4? or is it 2?
I think everywhere we are using the tab size of 2 .. so if it's 4, can you please change it to 2 and re-format this?

File: src/components/storyCard/storyCard.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (sumitd94) [inline src/components/storyCard/storyCard.stories.tsx:2]: instead of adding it here, let's do it in the eslint config file, please.

**Review by sumitd94 — CHANGES_REQUESTED** [review]
one small change and after that this looks good

File: src/components/storyCard/storyCard.stories.tsx (hunk 5 above)
Comment (correaswebert) [reply src/components/storyCard/storyCard.stories.tsx:2]: Also, I think for Next.js, we don't require the React import.... 😄

File: src/components/storyCard/index.tsx [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+            <span className={styles.dueDate}>{dueDate}</span>
+          </div>
+        </div>
+      </div>
+      <div className={styles.taskStatus}>
+        <div className={styles.statusInfo}>
+          Status:
+          {status === 'Active'
+            ? <span className={styles.subtaskStatus}>{status}</span>
+            : <img src="/lock-solid.svg" alt="lock-icon" height="30" />}
```
Comment (whyDontI) [inline src/components/storyCard/index.tsx:99]: Can you push this image please?

Comment (Cartmanishere) [reply src/components/storyCard/index.tsx:99]: Done

**Review by whyDontI — APPROVED** [review]

**Pull request merged**